src/ripple/nodestore/ShardSizeTuning.md

```diff
@@ -0,0 +1,212 @@
+# Shard Size Tuning
+
+The purpose of this document is to compare the sizes of shards containing
+varying amounts of ledgers.
+
+## Methodology
+
+One can see visually from a block explorer that typical mainnet ledger
```

Comment:
Should say "that **a** typical mainnet ledger"

Reply:
Done.